packages/adminrouter/extra/src/.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+# Ignore the following errors:
```

Comment:
I tried to be consistent with other repos, i.e. https://github.com/mesosphere/bouncer/blob/master/.flake8

From what I can see the difference is only in the max line length. 95 < 120, so AR code will definitely pass general DC/OS ones. On top of that - I ofter use my laptop and it has very narrow screen ;)

Would it be OK for you to have two sets of rules as long as the "local" ones are stricter than the DC/OS ones and not the other way round?